plugins/analytics-module-new-relic/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-analytics-module-newrelic-browser",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
```

Comment:
Don't believe this is the correct role.

I'm on mobile, so I don't have the correct value handy, but something like frontend module...

Reply:
Changed it to frontend-plugin-module as per [this doc](https://backstage.io/docs/local-dev/cli-build-system/#package-roles).